packages/mdc-select/_mixins.scss

```diff
@@ -218,3 +218,153 @@
     pointer-events: none;
   }
 }
+
+@mixin mdc-select-with-leading-icon_ {
+  @include mdc-select-icon-horizontal-position_(16px, 48px);
+
+  &.mdc-select--outlined {
+    @include mdc-floating-label-float-position($mdc-select-outlined-label-position-y, $mdc-select-outlined-with-leading-icon-label-position-x);
+    @include mdc-floating-label-shake-animation(text-field-outlined-leading-icon);
```

Comment:
This is currently assuming that text field is also installed and included. Should we replicate the shake keyframes for select, since I assume we're not currently making the jump to depending on text field?